packages/ts-moose-lib/src/scripts/workflow.ts

```diff
@@ -2,11 +2,10 @@ import { log as logger, proxyActivities } from "@temporalio/workflow";
 import { WorkflowState } from "./types";
 import { mooseJsonEncode } from "./serialization";
 
-const { executeScript, readDirectory } = proxyActivities({
-  startToCloseTimeout: "10 minutes",
+const { getActivityRetry, readDirectory } = proxyActivities({
+  startToCloseTimeout: "1 minutes",
   retry: {
-    // TODO: Use user's retry config
-    maximumAttempts: 3,
+    maximumAttempts: 1,
```

Comment:
Why is this set to 1?

Reply:
`const { getActivityRetry, readDirectory } = proxyActivities({`

These are our functions, not the user's task function.